packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -36,21 +38,36 @@ class EventDetails extends CoreElement {
     assert(tabNav != null);
     assert(content != null);
 
+    // The size of the event details section will change as the splitter is
+    // is moved. Observe resizing so that we can rebuild the flame chart canvas
+    // as necessary.
+    final observer = html.ResizeObserver(allowInterop((entries, _) {
```

Comment:
can you file a dart:html bug that this call to allowInterop is required?

Reply:
filed https://github.com/dart-lang/html/issues/102